### ISSUE TITLE
Investigate what happens to CLPs when the number of decimals for a token changes

### ISSUE THREAD

**Issue by pandaring2you** [Sifchain/sifnode#2573]
this was raised by @smartyalgo - token can change their decimal value after import - will this be an issue in the CLP logic?

**Comment by pandaring2you** [comment]
The solution for this is part of issues#1376 , incorporated under same timing estimate (2weeks)

**Comment by pandaring2you** [comment]
From @sheokapr :
I have started testing logic around decimal places - found earlier today that 0 decimal tokens are not added to the tokenregistry due to a constraint added in the sifnode code to avoid negative decimal values. Kevin/Alex fixed this issue and merged the change in the working branch. I have retested it and it is working fine now. I am doing a few other tests around decimals such as:

Creating Pools for tokens with decimals ranging from 0-255
Adding liquidity for tokens with decimals ranging from 0-255
Unbonding liquidity for tokens with decimals ranging from 0-255
Removing liquidity for tokens with decimals ranging from 0-255
Swaps (In and Out) for tokens with decimals ranging from 0-255
